cluster/conf/locators.template

```diff
@@ -14,15 +14,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
-# Specify the host name on which a Snappy lead will be started. Also
-# specify the startup directory where the logs and configuration files
-# for that lead instance are kept. Apart from these two things, snappy
-# specific properties can also be passed. If the directory and properties
-# are not specified a default directory is created inside the SPARK_HOME directory.
-#
-#     localhost    -dir=/tmp/data/locator (config args)
+# By default, SnappyData launch scripts will start a single locator on localhost
+# and uses <snappyData_dir>/work/localhost-locator-1/ as the directory for logs and 
+# statistics. 
+# Assuming your network is ssh enabled you can add hostnames (one line per host) to start
+# locator on multiple hosts. 
+#
+# Example configurations ....
+# I) Using bind address : By default, SnappyData binds to all interfaces. 
+# In Cloud environments with an internal IP and a different public IP address for 
+# clients, you should configure the bind-address
+#
+# localhost -client-bind-address=<public IP for clients>
```

Comment:
I added this sentence. But, I am not sure if this is correct. On AWS, if I am not mistaken, this needs to be the internal private IP address, not an elastic public IP. Amogh or someone needs to confirm.

Reply:
Can be either. If private IP is given here, user'll need to specify public IP as -J-Dgemfirexd.hostname-for-clients. I think this is covered in the AWS related section. @shyjaprabhu